examples/seismic/viscoacoustic/operators.py

```diff
@@ -0,0 +1,208 @@
+import sympy as sp
+import numpy as np
+
+from devito import (Eq, Operator, VectorTimeFunction, TimeFunction, NODE,
+                    div, grad, diag)
+from examples.seismic import PointSource, Receiver
+
+def viscoacoustic_blanch_symes(v, r, p, vp, irho, rho, t_s, tt, s, damp, **kwargs):
+    """
+    Stencil created from from Blanch and Symes (1995) / Dutta and Schuster (2014) 
+    viscoacoustic wave equation.
+
+    Parameters
+    ----------
+    v : VectorTimeFunction
+        The computed particle velocity.
```

Comment:
no need for "computed"

Reply:
Okay. It really isn't necessary. We will fix it.